core/src/main/java/io/aiven/klaw/service/AclControllerService.java

```diff
@@ -667,6 +667,22 @@ public ApiResponse approveAclRequests(String req_no)
       notifyUserType = ACL_REQUEST_FAILURE;
     } else {
       saveToTopicHistory(userDetails, tenantId, aclReq);
+      dbHandle.insertIntoActivityLog(
+          RequestEntityType.ACL.value,
+          tenantId,
+          aclReq.getRequestOperationType(),
+          aclReq.getTeamId(),
+          aclReq.getAcl_ip()
+              + "-"
+              + aclReq.getTopicname()
+              + "-"
+              + aclReq.getAcl_ssl()
```

Comment:
I think but can't be 100% sure it is normally either acl_ssl or acl_ip depending on which type of acl it is do we just need to add a tertiary stmt here to decide onthe correct one.